app/src/main/java/com/infomaniak/mail/data/models/Attachment.kt

```diff
@@ -0,0 +1,119 @@
+/*
+ * Infomaniak kMail - Android
+ * Copyright (C) 2022 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.mail.data.models
+
+import android.util.Log
+import com.google.gson.annotations.SerializedName
+import com.infomaniak.lib.core.networking.HttpClient
+import com.infomaniak.lib.core.networking.HttpUtils
+import com.infomaniak.mail.data.api.ApiRoutes
+import com.infomaniak.mail.data.cache.MailRealm
+import com.infomaniak.mail.utils.AccountUtils
+import com.infomaniak.mail.utils.KMailHttpClient
+import io.realm.MutableRealm.UpdatePolicy
+import io.realm.RealmObject
+import io.realm.annotations.PrimaryKey
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import okio.use
+import java.io.BufferedInputStream
+import java.io.File
+
+// @RealmClass(embedded = true) // TODO: https://github.com/realm/realm-kotlin/issues/551
+class Attachment : RealmObject {
+    @PrimaryKey // TODO: Remove `@PrimaryKey` when we have EmbeddedObjects
+    var uuid: String = ""
+
+    @SerializedName("part_id")
+    var partId: String = ""
+
+    @SerializedName("mime_type")
+    var mimeType: String = ""
+    var encoding: String = ""
+    var size: Int = 0
+    var name: String = ""
+
+    @SerializedName("disposition")
+    private var disposition: String? = null
+
+    @SerializedName("content_id")
```

Comment:
It is voluntary to put a line break at each SerializedName?

Reply:
Not really, it's the android studio formatting who imposes it